ecoscope/base/relocations.py

```diff
@@ -0,0 +1,215 @@
+import warnings
+from copy import deepcopy
+from functools import cached_property
+
+import numpy as np
+import pandas as pd
+from pyproj import Geod
+
+from ecoscope.base._dataclasses import (
+    RelocsCoordinateFilter,
+    RelocsDateRangeFilter,
+    RelocsDistFilter,
+    RelocsSpeedFilter,
+)
+from ecoscope.base.straightrack import StraightTrackProperties
+from ecoscope.base import EcoDataFrame
+
+
+class Relocations(EcoDataFrame):
+    """
+    Relocation is a model for a set of fixes from a given subject.
+    Because fixes are temporal, they can be ordered asc or desc. The additional_data dict can contain info
+    specific to the subject and relocations: name, type, region, sex etc. These values are applicable to all
+    fixes in the relocations array. If they vary, then they should be put into each fix's additional_data dict.
+    """
+
+    @classmethod
+    def from_gdf(cls, gdf, groupby_col=None, time_col="fixtime", uuid_col=None, **kwargs):
```

Comment:
```suggestion
    def from_gdf(cls, gdf, groupby_col=None, time_col="fixtime", uuid_col=None, copy: bool = True):
```
Since this is the only kwarg that's used in this method, it seems much clearer to name it explicitly.